docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,109 @@ You can follow these steps to validate all cluster update approaches.
+  Kubernetes distribution for upgrade guidance and follow the recommended upgrade paths.
+
+  - For PXK and PXK-E, refer to
+    [Upgrade Kubeadm Clusters](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
+  - For K3s, refer to [K3s Upgrades](https://docs.k3s.io/upgrades#version-specific-caveats)
+  - For RKE2, refer to [RKE2 Manual Upgrades](https://docs.rke2.io/upgrades/manual_upgrade)
+
+### Prerequisites
+
+- An active Kubernetes cluster in Palette.
+- A terminal client
+
+### Enablement
+
+The steps below show how we can use the second option while deploying the Hello Universe application pack and updating
+the cluster profile. In this example, we create an AWS cluster called cluster-update-deletion. We will use Palette and a
+terminal application.
+
+1. Log into Palette and create a cluster profile with two versions. The base layers should be ubuntu-aws, Kubernetes,
+   cni-calico, and csi-aws-ebs. The App Services layer will use the Hello Universe from the Palette Community Registry
+   (OCI).
+
+   The first profile version (1.0.0) should be the Hello Universe 1.2.0 pack with the API disabled.
+
+   The second version (1.1.0) should be the Hello Universe 1.2.0 pack with the API enabled.
+
+2. Build a cluster using the profile version 1.0.0 (takes about 15-20 min)
+3. Download the Admin kubeconfig file from the cluster Overview.
+
+   Sample output
+
+![How to download Admin kubeconfig file from Palette](/clusters_management_updates_kubeconfig-download.webp)
+
+4. Go your terminal application and run the following command
+
+`kubectl get pods --kubeconfig=admin.cluster-update-deletion.kubeconfig --namespace=hello-universe -o=custom-columns="POD_NAME:.metadata.name,CONTAINER_NAME:.status.containerStatuses[].name,CONTAINER_ID:.status.containerStatuses[].containerID"`
+
+5. This command will display the containers in the cluster that are in the hello universe namespace. It will include the
+   pod name, container name, and container ID. You can use these identifiers with various `kubectl` options.
+
+Sample output
+
+![Using kubectl to see what containers are running](/clusters_management_updates_kubectl-noapi-hellouniverse.webp)
+
+6. Return to Palette and update the cluster using the profile version 1.1.0 (takes about 1-2 min).
+7. Return to the terminal and run the following command
+
+`kubectl get pods --kubeconfig=admin.cluster-update-deletion.kubeconfig --namespace=hello-universe -o=custom-columns="POD_NAME:.metadata.name,CONTAINER_NAME:.status.containerStatuses[].name,CONTAINER_ID:.status.containerStatuses[].containerID"`
+
+Sample output
+
+![Using kubectl to see what containers are running in the updated cluster](/clusters_management_updates_kubectl-api-hellouniverse.webp)
+
+With the cluster updated, we can see that the previous version of the Hello Universe app remains. This container,
+however, is not used with the API version and is no longer needed.
```

Comment:
```suggestion
   With the cluster updated, we can see that the previous version of the Hello Universe app remains. This container,
   however, is not used with the API version and is no longer needed.
```

Reply:
The content within a numbered step must be indented otherwise it will be rendered outside of the numbered step.